frontend/src/components/authbuttons/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Button } from '@material-ui/core';
+import React from 'react';
+import { generatePath, useHistory } from 'react-router-dom';
+import { request } from '../../lib/k8s/apiProxy';
+import { getCluster, getClusterPrefixedPath } from '../../lib/util';
+import { ClusterDialog } from '../cluster/Chooser';
+import OauthPopup from '../oidcauth/OauthPopup';
+
+function AuthButtons(){
+  const history = useHistory();
+  const [authType, setAuthType] = React.useState<string | null>(null);
+  localStorage.setItem('selected_cluster', getCluster() as string);
```

Comment:
There should be a setter for this. If there's not please create it, otherwise it'll be difficult to maintain.

Reply:
ok